install_config/configuring_authentication.adoc

```diff
@@ -427,8 +427,16 @@ http://docs.openstack.org/developer/keystone/[Keystone] is an OpenStack project
 that provides identity, token, catalog, and policy services. You can integrate
 your {product-title} cluster with Keystone to enable shared authentication with
 an OpenStack Keystone v3 server configured to store users in an internal
-database. Once configured, this configuration allows users to log in to
-{product-title} with their Keystone credentials.
+database. This configuration allows users to log in to {product-title} with
+their Keystone credentials.
+
+You can configure the integration with Keystone so that the new {product-title}
+users are based on either the Keystone user names or unique Keystone IDs. With
+both methods, users log in
+by entering their Keystone user name and password. Basing
```

Comment:
No, the user never works with this data directly, it's completely internal stuff. Yeah, I think you are right - the main idea is to use Keystone ID instead of user name.
I think we can leave the text as it is.